db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+AncestorRef.create(
+  [
+    { service_name: 'web',        ref: 'production' },
+    { service_name: 'rs_west',    ref: '2b7a8338fde0a998d8aa6b540f1aa4dcb3f9018f' },
+    { service_name: 'rs_east',    ref: '2b7a8338fde0a998d8aa6b540f1aa4dcb3f9018f' },
+    { service_name: 'rs_central', ref: '2b7a8338fde0a998d8aa6b540f1aa4dcb3f9018f' }
```

Comment:
Setting the seeds as a sha that'll very quickly become stale doesn't feel great, we'll need to make sure we either 1) never use this to reconfigure the DB (and only use it for tests or 2) make sure we remember to update these if we ever do use it. (no immediate action needed from this comment, just a note)

Reply:
great point! I removed this creation from the seeds to avoid getting stale and moved the service creation to the migration file. Same comment as above, let me know if there's a better way to do that